wiki/Guides/Discord_Rich_Presence/id.md

```diff
@@ -0,0 +1,38 @@
+# Discord Rich Presence
+
+Sejak 31 Oktober 2017, Discord Rich Presence telah mencapai rilis yang stabil, dan bisa digunakan publik!
+
+## Apa artinya ini?
+
+Ini berarti, kamu sekarang dapat mengundang dan bergabung ke dalam multiplayer osu! dari dalam server Discord, kamu juga dapat melihat apa yang temanmu sedang mainkan di osu!, bahkan meminta untuk bergabung ke dalam multiplayer yang sedang berlangsung! Kamu juga dapat melihat (*spectate*) apa yang sedang dimainkan oleh orang-orang yang berada dalam server yang sama denganmu.
+
+## Bagaimana cara kerjanya?
+
+- **Pastikan untukd memperbarui dulu gim ke versi terbaru!** Fitur baru Discord yang mewah ini hanya berfungsi untuk versi gim mulai dari Halloween (2017) dan lebih baru - jika kamu melewati langkah ini maka tidak akan berfungsi.
+- Pastikan osu! telah terdaftar di sistem dan Discord-mu - biasanya kamu tidak perlu melakukan apa-apa jika sudah memasang, jadi tidak perlu khawatir. Kamu dapat memeriksanya lewat `App Settings` -> `Games` di menu pengaturan Discord.
+- Hidupkan opsi `Display currently running game as status message` di bagian `Games`. Langkah ini tidak diperlukan untuk bergabung dengan multiplayer yang diiklankan orang lain, tetapi jika kamu ingin mengiklankan ruang multiplayer-mu, lakukan.
+- Buat ruangan multiplayer baru, kunjungi server Discord favoritmu dan klik ikon ini ![tombol hijau mengkilap dengan simbol tambah](img/shiny-green-button.jpg). Jika belum muncul, tunggu beberapa detik jika kamu baru saja membuat ruangan multiplayer-nya.
+
+Berikut ditunjukkan dalam bentuk [video](https://assets.ppy.sh/media/halloween-2017/themoon.mp4).
+
+Jika kamu membuat ruangan multiplayer-mu dikunci dengan kata sandi, kamu tidak perlu memasukkan kata sandinya jika menggunakan fitur Discord ini.
+
+## Bergabung ke dalam multiplayer
+
+Kamu juga dapat mengklik profil Discord siapa pun yang bermain osu! dan satu server denganmu, dan melihat apa yang sedang mereka lakukan. Jika akunmu berteman dengannya, kamu bahkan dapat meminta untuk bergabung ke dalam multiplayer yang sedang berlangsung saat ini.
+
+![](img/join.jpg)
+
+## Melihat (*spectate*) pemain lain
+
+Tombol ini secara otomatis akan berubah juga fungsinya supaya bisa melihat (*spectate*) pemain lain yang sedang bermain.
```

Comment:
disini sebenernya ada 1 kalimat yang nggak ke translate: `so never miss out on your friends failing the same song repeatedly again!`

kira2, memang nggak perlu di translate nggak apa2 atau sengaja dilewatin karena blm dapet arti yang pas?

Reply:
Pesan dari Shurelia sih kalo trivial (ga terlalu berguna ato cuman tambahan), dan dirasa kata sebelumnya bisa berdiri sendiri aman bisa dihilangkan aja sekalian.